### PR TITLE
Implemented buildAutoChooser() in Python.

### PR DESCRIPTION
**Pull request by jserra99**
Was already doing this for my team's robotpy code so I figured why not implement it here. As far as my testing extends, it works, I am not 100% sure if I implemented the docs correctly, however.

### CODE REVIEW

**Comment by mjansen4857** [comment]
The documentation part should be changed to have tabs for different languages to be consistent with the rest of the examples. You should be able to copy one of those examples, remove the C++ tab, and change the code part. Let me know if you need any help with that.

**Comment by jserra99** [comment]
Done. Didn't catch that the first time around, was wondering how the switching worked.

File: Writerside/topics/pplib-Build-an-Auto.md [hunk 1]
```diff
@@ -471,4 +474,29 @@ public class RobotContainer {
...
+        SmartDashboard.putData("Auto Chooser", self.autoChooser)
+    
+    def getAutonomousCommand():
+        return self.autoChooser.getSelected()
+```
+
+</tab>
+</tabs>
+
+</snippet>
```
Comment (mjansen4857) [inline Writerside/topics/pplib-Build-an-Auto.md:502]: I believe this is a stray closing tag
```suggestion

```

**Review by mjansen4857 — CHANGES_REQUESTED** [review]

File: Writerside/topics/pplib-Build-an-Auto.md (hunk 1 above)
Comment (jserra99) [reply Writerside/topics/pplib-Build-an-Auto.md:502]: Is it? That was there before, I simply moved it down. The md originally opened with <snippet id="build-an-auto"> and ended with </snippet> immediately after the java example code.

Comment (mjansen4857) [reply Writerside/topics/pplib-Build-an-Auto.md:502]: Yeah you're right. Forgot that file is different

**Pull request merged**